cf.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 echo "------ Starting APP ------"
-gunicorn foia_hub.wsgi:application --log-file -
+if [ $CF_INSTANCE_INDEX = "0" ]; then
+    echo "----- Migrating Database -----"
+    python manage.py migrate --noinput
+    echo "----- Loading Agency Contacts -----"
+    python manage.py load_agency_contacts
+fi
+waitress-serve --port=$VCAP_APP_PORT foia_hub.wsgi:application
```

Comment:
Looks like this pull request doesn't just move the data loading and migrations to cf.sh. It also moves us from gunicorn to waitress. It's fine if that's your recommendation, but you should call that out. If we're going to move back to waitress, we're going to need to run a few more load tests and figure out our failover point.